exchanges/okx/okx.go

```diff
@@ -560,9 +560,6 @@ func (ok *Okx) AmendOrder(ctx context.Context, arg *AmendOrderRequestParams) (*O
 	if arg.ClientSuppliedOrderID == "" && arg.OrderID == "" {
 		return nil, errMissingClientOrderIDOrOrderID
 	}
-	if arg.NewQuantity <= 0 && arg.NewPrice <= 0 {
```

Comment:
This is not what I meant by [this](https://github.com/thrasher-corp/gocryptotrader/pull/1005#discussion_r1014916617) comment. This still has an issue where if the field is `0` it is still sent. Add an `omitempty` to the fields.